.github/workflows/ci.yml.jinja

```diff
@@ -31,7 +31,7 @@ jobs:
     needs: formatting
     strategy:
       matrix:
-        os: ['{{github_linux_image}}']
+        os: ['ubuntu-20.04']
         python:
           - version: '{{min_python}}'
             tox-env: 'py{{min_python|replace(".", "")}}'
```

Comment:
Still templated. There is a way to avoid this by reading the version from a file in a previous job, setting a variable and using here here, but that seemed overkill. Any better ideas? If not, there are very few actions in this workflow (and they are quite stable), so there may not be too many updates that would require manual handling.